apps/challenges/serializers.py

```diff
@@ -248,6 +248,10 @@ class Meta:
             "vpc_cidr",
             "subnet_1_cidr",
             "subnet_2_cidr",
+            "worker_instance_type",
+            "worker_ami_type",
+            "worker_disk_size",
+            "max_worker_instance",
```

Comment:
Can we please add `max_worker_instance` and `desired_worker_instance` here as well.